### ISSUE TITLE
Where to download the "vision-xxx.tar.gz"?

### ISSUE THREAD

**Issue by bad-meets-joke** [lancercat/VSDF#6]
Could you share the link to download the torchvision code, i.e. vision-xxx.tar.gz described in manul.pdf,  of which the version is also compatible with pytorch1.7.1+cu10.1?

**Comment by lancercat** [comment]
TLDR: Sorry, I dunno. Please develop on up-to-date enviornments(torch 112, vision 0.13).

--------------------------------- 

Longer story
Ugh, the vision part on the ablative is not installed by recent, so I don't have the installation packages either.

And the guide is intended to set up an up-to-date environment but not the exact same one with my ablative machine.
The machine has more than one users so I can't simply update it on my own...

My suggestion is to update everything, specifically using pytorch 112 and vision 0.13.

**Comment by lancercat** [comment]
FYI, the vision on the ablative machine has version string '0.8.2+cu101'
![image](https://user-images.githubusercontent.com/59994105/194759683-a70d9232-ed6a-4adb-bd33-928487b02776.png)

Still, I highly recommend to develop on up-to-date environment, which  is more future proof.

**Comment by bad-meets-joke** [comment]
> TLDR: Sorry, I dunno. Please develop on up-to-date enviornments(torch 112, vision 0.13).
> 
> Longer story Ugh, the vision part on the ablative is not installed by recent, so I don't have the installation packages either.
> 
> And the guide is intended to set up an up-to-date environment but not the exact same one with my ablative machine. The machine has more than one users so I can't simply update it on my own...
> 
> My suggestion is to update everything, specifically using pytorch 112 and vision 0.13.

Ok，I will change to pytorch1.12.0, so how to get the vision-0.13.tar.gz as decribed in the manual.pdf?

**Comment by lancercat** [comment]
you can get it from the official site  :-)
They seem to have fixed the ffmpeg bug.

https://github.com/pytorch/vision/releases

Alternatively you can find it in
https://github.com/lancercat/make_env 

they should be the same.

**Comment by bad-meets-joke** [comment]
Thanks, I run on the Pytorch1.7.1+cu10.1 due to the limitation of Nvidia Driver version, and directly use `conda install torchvision==0.8.2`, it finally works.

**Comment by lancercat** [comment]
Thanks for your feedback! I pinned this issue as I think pretty many people using cent or ubuntu may find this helpful.
